projects/user-telemetry-client/yarn.lock

```diff
@@ -0,0 +1,18 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
We use npm for other projects in this repo – we should use npm here to be consistent, and if we'd like to convert this to yarn (which would be welcome!), that should be another PR.

Reply:
This will take some amount of teamcity retooling unfortunately, so I might. (Unless we go for manual publication)